paddlecloud/paddlejob/views.py

```diff
@@ -359,3 +359,36 @@ def post(self, request, format=None):
                 fn.write(data)
 
         return Response({"msg": ""})
+
+
+class SimpleFileList(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+    parser_classes = (FormParser, MultiPartParser,)
+
+    def get(self, request, format=None):
+        """
+        Simple list files.
+        """
+        file_path = request.query_params.get("path")
+        dc = request.query_params.get("dc")
+        # validate list path must be under user's dir
+        path_parts = file_path.split(os.path.sep)
+        msg = ""
+        if len(path_parts) <= 1:
```

Comment:
Can we use a regex instead of so much `if else`? But it's not blocking the merge. LGTM!